includes/admin/welcome.php

```diff
@@ -154,9 +154,9 @@ public function about_screen() {
 
             <p class="about-text"><?php
 				printf(
-				/* translators: %s: https://givewp.com/documenation/ */
+				/* translators: %s: https://givewp.com/documentation/ */
```

Comment:
We should instead use our shortlink: http://docs.givewp.com/docs

Reply:
Fixed : https://github.com/WordImpress/Give/commit/365431e71d9aae74475b14ca5ff152870d6f606b